management-entity/src/test/java/org/terracotta/management/entity/ManagementEntityTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.management.entity;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.terracotta.connection.Connection;
+import org.terracotta.connection.entity.EntityRef;
+import org.terracotta.exception.EntityAlreadyExistsException;
+import org.terracotta.exception.EntityNotFoundException;
+import org.terracotta.exception.EntityNotProvidedException;
+import org.terracotta.exception.EntityVersionMismatchException;
+import org.terracotta.management.entity.client.ManagementAgentEntity;
+import org.terracotta.management.entity.client.ManagementAgentEntityClientService;
+import org.terracotta.management.entity.server.ManagementAgentEntityServerService;
+import org.terracotta.management.model.context.ContextContainer;
+import org.terracotta.management.registry.AbstractManagementRegistry;
+import org.terracotta.management.registry.ManagementRegistry;
+import org.terracotta.passthrough.IClusterControl;
+import org.terracotta.passthrough.PassthroughServer;
+import org.terracotta.passthrough.PassthroughTestHelpers;
+
+/**
+ * @author Mathieu Carbou
+ */
+@RunWith(JUnit4.class)
+public class ManagementEntityTest {
+
+  @Test
+  public void test_expose() throws EntityNotProvidedException, EntityVersionMismatchException, EntityAlreadyExistsException, EntityNotFoundException {
```

Comment:
not sure to understand what's the assertion made here ?
it's just a smoke test ?